src/test/kotlin/unitTest/tasks/CheckWifiTaskUnitTest.kt

```diff
@@ -29,13 +29,16 @@ class CheckWifiTaskUnitTest : BaseUnitTest() {
     val bridge: AndroidDebugBridge = mock()
     val outputReceiver: CollectingOutputReceiver = mock()
     val outputReceiverProvider: OutputReceiverProvider = mock()
-    val defaultPluginTask: DefaultPluginTask = mock()
 
     val deviceCommunicator = DeviceCommunicator(bridge, outputReceiverProvider)
-    val noDevices = emptyArray<IDevice>()
-    val wifi = "wifi"
+    val wifi = "wlanName"
     val devices = arrayOf(device)
     val mNetworkInfo = "mNetworkInfo [type: WIFI[], extra: \"$wifi\"]"
+    val wifiDumpsysAndroid9 = "mWifiInfo SSID: $wifi, BSSID: 50:06:04:c2:f5:5d, MAC: a8:db:03:e0:e1:e5, Supplicant state: COMPLETED, RSSI: -62, Link speed: 270Mbps, Frequency: 5180MHz, Net ID: 2, Metered hint: false, GigaAp: false, VenueName: null, WifiMode: 4, score: 60\n" +
```

Comment:
is this real data? Should we pass this information (BSSID, MAC, ...) to the users of our plugin? Let's create fake data and use original values only for the relevant parts.